### PR TITLE
Fix #12 add ipynb template

### LINKED ISSUE #12: ipynb version of the template
Probably a quarto-converted version is enough.

### PR DESCRIPTION
**Pull request by chainsawriot**

### CODE REVIEW

**Comment by chainsawriot** [comment]
@rgaiacs Could you review this, especially the missing features?

File: readme.md [hunk 1]
```diff
@@ -129,3 +129,12 @@ mean(mtcars$mpg)
 plot(mtcars$mpg, mtcars$wt)
 ```
 ````
+
+## Missing features of Jupyter notebook
+
+Jupyter notebooks are rendered into HTML with quarto using the Jupyter kernel. Comparing to native quarto documents, the following features are not supported.
+
+* Built-in YAML Metadata
```
Comment (rgaiacs) [inline readme.md]: The YAML header must be in the first cell that must be of type `raw`. See https://quarto.org/docs/tools/jupyter-lab.html#yaml-front-matter.

**Review by rgaiacs — CHANGES_REQUESTED** [review]
Looks good. Minor correction on YAML header.

File: readme.md (hunk 1 above)
Comment (chainsawriot) [reply readme.md]: Thank you very much! It should been fixed.

**Pull request merged**